src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -4,66 +4,51 @@
 import java.util.List;
 
 public class InputFieldAnnotations {
-    public @interface SomeAnnotation1 { }
+  public @interface SomeAnnotation1 {}
 
-    public @interface SomeAnnotation2 { }
+  public @interface SomeAnnotation2 {}
 
-    public @interface SomeAnnotation3 {
-        int x();
-    }
+  public @interface SomeAnnotation3 {
+    int x();
+  }
 
-    /**
-     * testing.
-     */
-    @SomeAnnotation1
-    @SomeAnnotation2
-    String name = "Zops";
```

Comment:
please restore all annotation related formatting.
https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations

>Annotations applying to a field also appear immediately after the documentation block, but in this case, multiple annotations (possibly parameterized) may be listed on the same line;

may be, but does not have to.
User is ok to do formatting like this , if user use formatter and it make it single line, ok it is his choice, we just do not need to make violation on what formatter is doing, as both approaches are valid.

Reply:
done